portal/common/lib/constants.ts

```diff
@@ -14,3 +14,11 @@ export const SITE_NAMES: { [key: string]: string } = {
 export const FALLBACK_PORTAL = "blob.store";
 // The string representing the ResourcePath struct in the walrus_site package.
 export const RESOURCE_PATH_MOVE_TYPE = SITE_PACKAGE + "::site::ResourcePath";
+
+export const TESTNET_RPC_LIST = [
+    'https://fullnode.testnet.sui.io:443',
+    'https://sui-testnet.public.blastapi.io/',
+    'https://sui-testnet-endpoint.blockvision.org/',
+    'https://rpc.ankr.com/sui_testnet',
+    'https://sui.blockpi.network/v1/rpc/public'
```

Comment:
Nit: use a common format for the urls. I.e., all have/don't have the trailing `/`.